### DISCUSSION TITLE
Disable auth for specific apps

### DISCUSSION THREAD

**Discussion by SomniVertix** [Budibase/budibase#7706] [Help] [answered]
How can I disable auth for specific apps - there's some apps that we want as a page you can go to without logging in since they're pretty much open information pages

**Comment by melohagan** [comment, marked as answer]
Hey @SomniVertix 

You can choose which screens are **public** which means no login will be required to view them. More here: https://docs.budibase.com/docs/screens#access-level

**Reply by SomniVertix** [reply]
Ah, thank you! That worked
